deployment/playbooks/includes/ansible_venv.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: "Raspbian-specific ansible venv packages"
+  pip:
+    virtualenv: "{{ my_ansible_vitualenv }}"
+    virtualenv_command: /usr/bin/python3 -m venv
+    name:
+      - cython
+  when: ansible_lsb.id == "Raspbian"
+
+# Use anisible interpreter to separate the volttron requirements from
+# the requirements of the volttron instance
+- name: Create ansbile virtual env
+  pip:
+    virtualenv: "{{ my_ansible_vitualenv }}"
+    virtualenv_command: /usr/bin/python3 -m venv
+    name:
+     - pyyaml
+     - psutil
+     - zmq
+     - 'netifaces==0.10.9'
+     - 'deepdiff'
+
```

Comment:
Why do we need to install these packages in ansible virtual environment?

Reply:
They are required...